certificate-policy.md

```diff
@@ -168,42 +169,44 @@ The key words "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMME
 
 
 # 2. PUBLICATION AND REPOSITORY RESPONSIBILITIES
-The FPKIPA will review and update this Certificate Policy at least every 365 days to ensure compliance with CAB Forum Baseline Requirements. After review and approval, the CP document version number and a dated changelog entry shall be added even if no changes were deemed necessary.
-
-The review and update shall include any changes needed to address:
-
-- U.S. Federal Government mission needs and changes to procedures to support the missions
-- CAB Forum Baseline Requirements updates
-
-If changes to CAB Forum Baseline Requirements are made and have applicable requirements which require compliance earlier than 365 days from the last update, the FPKIPA will update the policy to meet those compliance timeframes and ensure all CA's and associated CA Certification Practice Statements are updated.  
-
-Each CA shall develop, implement, enforce, and update at least every 365 days a Certification Practice Statement (CPS) that describes in detail how the CA implements the requirements of this CP.
-
-An annual self-assessment shall be conducted by any CA operating under this Certificate Policy and the accompanying CA CPS to show compliance with the latest version of this certificate policy and the CAB Forum Baseline Requirements. The CA shall indicate the self-assessment by incrementing the CPS version number and adding a dated changelog entry to the change record.
 
 ## 2.1 Repositories
-The CA shall make revocation information for Subordinate Certificates and Subscriber Certificates available in accordance with this Policy.
+Each CA shall disclose the following information through a publicly accessible Repository: 
+- CA Certificates
+- Certificate Revocation Lists (CRLs)
+- Online Certificate Status Protocol responses
+- CPS documents
+- Subscriber Agreements
+- Audit Reports
```

Comment:
I think it would help to be explicit that these should be unredacted.

Reply:
addressed